src/hooks/useAffirmation.js

```diff
@@ -1,3 +1,27 @@
-export default function useAffirmation() {
-  // TODO: Implement me!
+import { useEffect, useState } from 'react';
+
+
+export default function useAffirmations() {
+ //naming and defining the useState
+  const [affirmation, setAffirmation] = useState('');
+  const [fgColor, setFgColor] = useState('#ffcc00');
+ const [bgColor, setBgColor] = useState('#212121');
```

Comment:
these will be a separate instance of state here, so your useEffect won't track them as intended (because they're not being changed anywhere). instead, you'll want to bring `fgColor` and `bgColor` into your hook as parameters and then pass them in when you call your hook